Packs/Legacy/Integrations/PhishLabsIOC/PhishLabsIOC.py

```diff
@@ -587,28 +588,47 @@ def get_feed_request(since: str = None, limit: str = None, indicator: list = Non
     return response
 
 
+def get_sec_time_delta(last_fetch_time):
+    # try in UTC first
+    fetch_delta = datetime.utcnow() - last_fetch_time
+    fetch_delta_in_sec = fetch_delta.seconds
+
+    # if negative then try in current time
+    if fetch_delta_in_sec < 0:
+        fetch_delta = datetime.now() - last_fetch_time
+        fetch_delta_in_sec = fetch_delta.seconds
+
+    # if negative default to 1 day
+    if fetch_delta_in_sec < 0:
+        fetch_delta_in_sec = SEC_IN_DAY
+
+    return str(fetch_delta_in_sec) + "s"
+
+
 def fetch_incidents():
     """
     Fetches incidents from the PhishLabs user feed.
     :return: Demisto incidents
     """
     last_run: dict = demisto.getLastRun()
     last_fetch: str = last_run.get('time', '') if last_run else ''
-    last_offset: str = last_run.get('offset', '0') if last_run else '0'
+    last_fetch_time: datetime = (datetime.strptime(last_fetch, '%Y-%m-%dT%H:%M:%SZ') if last_fetch
```

Comment:
Seems that you don't have last fetch you will go to NONE_DATE and will never use FETCH_TIME am i missing something?

Reply:
FETCH_TIME is how far back in time to fetch for the first time.
NON_DATE is a base date = '0001-01-01T00:00:00Z' - to be saved in ***last_fetch_time*** on the first fetch (that is if ***last_fetch*** is None).

The first fetch happens in line 621 of the code in case ***last_fetch*** is none - there you can see it uses FETCH_TIME to bring the first batch of incidents.

***last_fetch_time*** is used later to make sure you only bring back new incidents (line 639) but this is irrelevant for the first fetch and only comes into play on second fetch.